src/reducers/index.js

```diff
@@ -9,8 +9,10 @@ export default (count = 0, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+    return action.payload + 1;
```

Comment:
We have the `count` available to us, and all we need to do is to increment and decrement. `action.payload` is unnecessary, we can simply `++count` and `--count`.